docs/sdk/migrations/datil.md

```diff
@@ -0,0 +1,176 @@
+
+Previously Lit networks were powered by a pre-Bedrock Optimism L2 rollup called [Chronicle](../../network/rollup.mdx). In our effort to deliver a more performant and stable backend for Lit's infrastructure, we're launching a new network, Vesuvius, powered by [Arbitrum Orbit](https://arbitrum.io/orbit).
+
+Depending on the existing Lit network you are using, you should migrate to it's corresponding Datil network:
+
+| Currently Available | Requires Payment | Your Current Network | Network to Migrate to | Description                                                 |
+| --------------------| ---------------- | -------------------- | -------------------- | ------------------------------------------------------------ |
+|          ❌         |          ✅      | `habanero`           | `datil`               | Decentralized mainnet designed for production use cases      |
+|          ❌         |          ✅      | `manzano`            | `datil-test`          | Decentralized testnet designed for pre-production deployment |
+|          ✅         |          ❌      | `cayenne`            | `datil-dev`           | Centralized testnet designed for early-stage development     |
+
+Like their counterparts, `datil` and `datil-test` require payment for usage of the Lit network via [Capacity Credits](../capacity-credits.md); however, `datil-dev` does not.
+
+## Breaking Changes and Important Updates
+
+### Datil Resources
+
+Datil has a new [block explorer](https://vesuvius-explorer.litprotocol.com/), and Lit test tokens can be received from [this faucet](https://datil-dev-faucet.vercel.app/).
+
+### New Network, New PKPs
+
+Because existing PKPs were minted on the old Chronicle chain, migration from `cayenne` to `datil-dev` requires re-minting of all PKPs. This also means transferring ownership of assets owned by PKPs minted on Chronicle, to the newly minted ones on Chronicle Vesuvius.
+
+To reduce the friction of migrating PKPs to Datil, we've written a [migration script](https://github.com/LIT-Protocol/developer-guides-code/tree/wyatt/pkp-migration-script/pkp-migration/nodejs) that will take a list of PKP public keys, fetch their configured Auth Methods and Scopes, and mint new PKPs on a target Lit Network, setting the same Auth Methods and Scopes for each PKP.
+
+:::note
+The migration script **will not** handle migration of any assets the existing PKPs own such as tokens.
+
+Additionally, the newly minted PKPs on the target Lit network will have new Ethereum addresses, so anything that uses the existing PKP's Ethereum address for things like permissions, will need to be manually updated to us the new PKP's Ethereum address.
+:::
+
+### Encrypted Data
+
+Because Vesuvius is a new Lit network, a new round of Distributed Key Generation (DKG) has been done, resulting in new BLS root keys for the network. This means all previously encrypted data using `cayenne`'s public BLS key, will **not** be able to be decrypted using the `datil-dev` network.
```

Comment:
Should we generalize this? So instead of saying 'encrypted using 'cayenne's' BLS key' we can just say that any data encrypted on a previous Lit network won't be able to be decrypted on the new network

Reply:
Updated via [this commit](https://github.com/LIT-Protocol/docs/pull/306/commits/fdfacaf3ea348a5fd5bdd85d0a52cb9355ee7fe1)